packages/tables/backend/sirius-components-tables/src/main/java/org/eclipse/sirius/components/tables/components/LineComponent.java

```diff
@@ -73,30 +74,46 @@ public Element render() {
     }
 
     private Element doRender(VariableManager lineVariableManager, String targetObjectId, Optional<Line> optionalPreviousLine) {
-        LineDescription lineDescription = this.props.getLineDescription();
-        UUID lineId = optionalPreviousLine.map(Line::getId).orElseGet(() -> this.computeLineId(targetObjectId));
+        LineDescription lineDescription = this.props.lineDescription();
+        UUID rowId = optionalPreviousLine.map(Line::getId).orElseGet(() -> this.computeLineId(targetObjectId));
 
         String targetObjectKind = lineDescription.getTargetObjectKindProvider().apply(lineVariableManager);
 
-        var cells = this.getCells(lineVariableManager, lineId);
+        var cells = this.getCells(lineVariableManager, rowId);
+        boolean resizable = lineDescription.getIsResizablePredicate().test(lineVariableManager);
+        Integer initialHeight = lineDescription.getInitialHeightProvider().apply(lineVariableManager);
```

Comment:
int

Reply:
Unless I miss something `initialHeightProvider` should use `Integer` as the returned type since no primitive types are allowed in `Function<>`